projects/packages/sync/src/replicastore/class-table-checksum-user.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Table Checksums Class.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+namespace Automattic\Jetpack\Sync\Replicastore;
+
+/**
+ * Class to handle Table Checksums for the User Meta table.
```

Comment:
I think should be  `Users` table.

```suggestion
 * Class to handle Table Checksums for the Users table.
```